zipkin-server/src/main/java/zipkin2/server/internal/ZipkinQueryApiV2.java

```diff
@@ -130,6 +131,19 @@ public String getTrace(@PathVariable String traceIdHex, WebRequest request) thro
     return new String(SpanBytesEncoder.JSON_V2.encodeList(trace), UTF_8);
   }
 
+  @GetMapping(
+    value = "/tagKeys",
+    produces = APPLICATION_JSON_VALUE)
+  public List getTagKeys() throws IOException {
```

Comment:
I think the type of return value should be `List<String>` or `ResponseEntity<List<String>>`.

Reply:
Fixed